web/server/entrypoint_arm64.go

```diff
@@ -0,0 +1,27 @@
+//go:build cgo
```

Comment:
Because it's in the filename. `_arm64.go` ensures it only builds for that arch.

Reply:
Added build tags to leave the android pathway as is. I used the `android` build tag rather than `no_cgo` since it's clearer, and both appear to only be used for `server-android` target. Let me know if there's a reason I should use the `no_cgo` tag instead.